draft-ietf-rtcweb-jsep.xml

```diff
@@ -2089,9 +2089,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               direction:
```

Comment:
No, we'd decided track IDs only go in sendrecv or sendonly descriptions. The "track" event of RTCPeerConnection only fires when applying a media description with a "send" direction, so that's the only place where a track ID is needed.

The reasoning was: if an application is using `addTrack`, and they send an offer and receive a "recvonly" answer, because the answerer hasn't called `addTrack` yet, the "track" event shouldn't fire for the offerer.